azure-pipelines.sds.qa-release.yml

```diff
@@ -0,0 +1,149 @@
+resources:
+  repositories:
+    - repository: azTemplates
+      type: github
+      name: hmcts/azure-devops-templates
+      ref: refs/heads/azure-dns-updates
```

Comment:
should this not be master?

Reply:
you have reminded me I need to make a PR for this branch, thanks!